boards/FSM/Test/Src/Test_WheelSpeedSensor.cpp

```diff
@@ -0,0 +1,67 @@
+#include "Test_Fsm.h"
```

Comment:
This test is unnecssary just like https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/582/files#r421798767

If you want to check for wheel speed can message you should be doing that in Test_StateMachine